include/aws/crt/io/TlsOptions.h

```diff
@@ -101,10 +102,11 @@ namespace Aws
                  */
                 void OverrideDefaultTrustStore(const char *caPath, const char *caFile) noexcept;
 
+                void OverrideDefaultTrustStore(const ByteCursor &ca) noexcept;
+
               private:
                 aws_tls_ctx_options m_options;
-
-                TlsContextOptions() noexcept;
+                bool m_isInit;
```

Comment:
The other way to do this (I hate flags like this, but that might just be me) is:

```cpp
aws_tls_ctx_options m_options_storage;
aws_tls_ctx_options *m_options = nullptr;

// and then when it's time to initialize
m_options = &m_options_storage;

// and then in the dtor:
if (m_options)
     aws_tls_ctx_options_clean_up(m_options);
```